spec/features/user_views_past_orders_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+RSpec.describe "User views past orders" do
+  context "with valid authorization" do
+    let(:user) { User.create(full_name: "example",
+                             email: "example@example.com",
+                             password: "password"
+                             )
+               }
```

Comment:
`end` at 9, 15 is not aligned with `let(:user) { User.create(full_name: "example",` at 5, 4